ansible/roles/flannel/handlers/main.yml

```diff
@@ -14,8 +35,21 @@
   command: ip link delete docker0
   when: '"docker0" in ansible_interfaces'
 
-- name: start docker
+  # TODO: work only on default installation...
+- name: docker graph driver 
+  command: findmnt -no FSTYPE -T /var/lib/docker
```

Comment:
I don't want to duplicate extra stuff from the docker role in flannel. Can we just leave the network stuff here and not do the BTRFS stuff?  Do we need to do the docker0 stuff in the docker role?